modules/apps/headless/headless-asset-library/headless-asset-library-test/src/testIntegration/java/com/liferay/headless/asset/library/resource/v1_0/test/AssetLibraryResourceTest.java

```diff
@@ -68,6 +76,58 @@ public void testDeleteAssetLibraryLinkToSite() throws Exception {
 				assetLibrary.getLinkedSitesExternalReferenceCodes()));
 	}
 
+	@Override
+	@Test
+	public void testDeleteAssetLibraryUserAccountUser() throws Exception {
+		AssetLibrary assetLibrary = _addAssetLibrary();
+
+		Integer initialUsersCount = assetLibrary.getUsersCount();
+
+		User user1 = UserTestUtil.addUser();
+		User user2 = UserTestUtil.addUser();
+
+		assetLibraryResource.postAssetLibraryUserAccountUser(
+			assetLibrary.getId(), user1.getUserId());
+		assetLibraryResource.postAssetLibraryUserAccountUser(
+			assetLibrary.getId(), user2.getUserId());
+
+		assetLibrary = assetLibraryResource.deleteAssetLibraryUserAccountUser(
+			assetLibrary.getId(), user1.getUserId());
+
+		Assert.assertTrue(
```

Comment:
@robertoDiaz, better use assertEquals here. And in other places where you AssertTrue with an ==

Reply:
Hi @jkappler , 

AssertEquals doesn't work well with Integers, this is why I used assertTrue

Reply:
In any case, I've updated it